common/src/main/resources/plugin.yml

```diff
@@ -6,12 +6,11 @@ author: "The BedwarsRel-Team | www.github.com/orgs/BedwarsRel/people"
 website: "www.github.com/BedwarsRel"
 
 main: io.github.bedwarsrel.BedwarsRel
-version: ${versionstring}
+version: ${full.version}
```

Comment:
This is needed because of dynamic versioning via Maven build profiles

Reply:
Yeah, I just did not know it was a Maven project.